ocp_utilities/infra.py

```diff
@@ -41,3 +46,133 @@ def assert_nodes_schedulable(nodes):
         raise NodeUnschedulableError(
             f"Following nodes are in unscheduled state: {unschedulable_nodes}"
         )
+
+
+class DynamicClassCreator:
+    """
+    Taken from https://stackoverflow.com/a/66815839
+    """
+
+    def __init__(self):
+        self.created_classes = {}
+
+    def __call__(self, base_class):
+        if base_class in self.created_classes:
+            return self.created_classes[base_class]
+
+        class BaseResource(base_class):
+            def __init__(self, *args, **kwargs):
+                super().__init__(*args, **kwargs)
+
+            @staticmethod
+            def _set_cnv_tests_label(res):
```

Comment:
we need to handle this `created-by-cnv-tests`, we cannot leave it bounded to cnv-tests

Reply:
done